python/lsst/afw/geom/skyWcs/skyWcs.cc

```diff
@@ -149,6 +150,10 @@ PYBIND11_MODULE(skyWcs, mod) {
     // Do not wrap readStream or writeStream because C++ streams are not easy to wrap.
     cls.def_static("readString", &SkyWcs::readString);
     cls.def("writeString", &SkyWcs::writeString);
+
+    utils::python::addOutputOp(cls, "__str__");
+    // Alternative repr: call writeString for the full AST output (which is very long).
```

Comment:
The comment's a bit unclear, given that this seems to *not* be what you're doing. Maybe something like "We could implement repr using writeString instead of operator<<..."?